src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourceArray = sourceString.split(';');
```

Comment:
Empty strings left after splitting with `;` can create unnecessary elements.
Example solution for  AI comment: 
```suggestion
  const sourceArray = sourceString.split(';').filter(property => property.trim() !== '');
```